README.md

```diff
@@ -6,52 +6,64 @@
+
 ### With Serverless framework:
-* Install the [**serverless-lumigo-plugin**](https://github.com/lumigo-io/serverless-lumigo-plugin/blob/master/README.md)
+* To configure Serverless framework, simply install our plugin: [**serverless-lumigo-plugin**](https://github.com/lumigo-io/serverless-lumigo-plugin/blob/master/README.md)
 
 ### Manually:
-Install `@lumigo/tracer`:
+
+To manually configure Lumigo in your Lambda functions:
+
+* First, install the `@lumigo/tracer` package using NPM:
 
  npm: 
 ~~~bash
 $ npm i @lumigo/tracer
 ~~~
     
-Wrap your `handler` (replace `DEADBEEF` with your token):
+* Next, wrap your `handler` in lumigo's `trace` function (note: replace `YOUR-TOKEN-HERE` with your Lumigo API token):
 
 ~~~js
-const lumigo = require('@lumigo/tracer')({ token: 'DEADBEEF' })
+const lumigo = require('@lumigo/tracer')({ token: 'YOUR-TOKEN-HERE' })
 
 const myHandler = async (event, context, callback) => { ... }
 
 exports.handler = lumigo.trace(myHandler)
 ~~~
 
+* Your function is now fully instrumented
+
 ## Configuration
-* You can turn on the debug logs by setting the environment variable `LUMIGO_DEBUG=TRUE`
-* You can prevent lumigo from sending keys that answer specific regexes by defining `LUMIGO_SECRET_MASKING_REGEX=["regex1", "regex2"]`. By default, we use the default regexes `[".*pass.*", ".*key.*", ".*secret.*", ".*credential.*", ".*passphrase.*"]`. All the regexes are case-insensitive.
-* Similarly, you can prevent lumigo from sending the entire headers and body of specific domains using the environment variable LUMIGO_DOMAINS_SCRUBBER=[".*secret.*"] (give it a list which is a json parsable). By default, we will use ["secretsmanager\..*\.amazonaws\.com", "ssm\..*\.amazonaws\.com", "kms\..*\.amazonaws\.com"].
-Note that if you do specify a domains list - the default list will be overridden.
-* In case of need, there is a kill switch, that stops all the interventions of lumigo immediately, without changing the code. Simply add an environment variable `LUMIGO_SWITCH_OFF=TRUE`.
+`@lumigo/tracer` offers several different configuration options. Pass these to the Lambda function as environment variables:
+
+* `LUMIGO_DEBUG=TRUE` - Enables debug logging
+* `LUMIGO_SECRET_MASKING_REGEX=["regex1", "regex2"]` - Prevents Lumigo from sending keys that match the supplied regular expressions. All regular expressions are case-insensitive. By default, Lumigo applies the following regular expressions: `[".*pass.*", ".*key.*", ".*secret.*", ".*credential.*", ".*passphrase.*"]`. 
+* `LUMIGO_DOMAINS_SCRUBBER=[".*secret.*"]` - Prevents Lumigo from sending the header and body from a specific domain. This accepts a comma-separated list of regular expressions that is JSON-formatted. By default, the tracer uses `["secretsmanager\..*\.amazonaws\.com", "ssm\..*\.amazonaws\.com", "kms\..*\.amazonaws\.com"]`. **Note** - These defaults are overridden when you define a different list of regular expressions.
```

Comment:
Collecting any data related to these domains, also response details.

Reply:
Added additional text to clarify this!

Reply:
@doriaviram @efimk-lu 
Can we include other attributes like "headers", "body", "event", "envs" ?